qa/tasks/cephfs/test_misc.py

```diff
@@ -198,6 +198,16 @@ def test_dump_inode_hexademical(self):
         info = self.fs.mds_asok(['dump', 'inode', hex(ino)])
         assert info['path'] == "/foo"
 
+    def test_fs_lsflags(self):
+        self.fs.set_joinable(False)
+        self.fs.set_allow_new_snaps(False)
+        self.fs.set_allow_multimds_snaps(False)
+        self.fs.set_allow_standby_replay(True)
+        lsflags = json.loads(self.fs.mon_manager.raw_cluster_cmd('fs', 'lsflags', self.fs.name, "--format=json-pretty"))
+        self.assertEqual(lsflags["not_joinable"], True)
```

Comment:
@batrick `not_joinable` is displayed/used as `joinable` also. 
See the command `ceph fs set cephfs joinable`, which updates `not_joinable` value also. So why there are two representations for the same thing?

Reply:
Because it's confusing to have a command like:

`ceph fs set cephfs notjoinable X`